source/css/_common/components/post/post.styl

```diff
@@ -46,7 +46,6 @@
 
 @import "post-expand";
 @import "post-collapse";
-@import "post-type";
```

Comment:
unneeded?

Reply:
`.post-type-quote` in `post-type.styl` no longer exist, so removed it

Reply:
https://github.com/theme-next/hexo-theme-next/blob/243804ad75ec4ff146ed7225c46264e72cbe8e8b/layout/_macro/post.swig#L37-L38